spec/mobile-fu/tablet_spec.rb

```diff
@@ -7,7 +7,7 @@
     ['Ipad', 
      'ipad', 
      'IPAD',
-     'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; ARM; Trident/6.0; Touch)',
+     'Mozilla/5.0 (Linux; Android 4.4; SM-T700 Build/KOT49H)',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.